airflow/jobs.py

```diff
@@ -343,61 +343,96 @@ def import_errors(self, dagbag):
                 filename=filename, stacktrace=stacktrace))
         session.commit()
 
+    def _active_runs(self, session, dag):
```

Comment:
Shouldn't this be a public dag property? You can replicate the @provide_session pattern as well